packages/cli/src/cli-commands/__tests__/cli-version-commands.spec.ts

```diff
@@ -2,6 +2,9 @@ import { describe, expect, it, vi } from 'vitest';
 
 vi.mock('@lerna-lite/version', () => null);
```

Comment:
ok I got all your tests working by changing the mock, so change the line 3
```ts
vi.mock('@lerna-lite/version', () => null);
```

with this and that should work
```ts
vi.mock('@lerna-lite/version', () => {
  const VersionCommand = null;
  return { VersionCommand };
});
```
![image](https://user-images.githubusercontent.com/643976/232967064-679dfb8d-7882-4868-90ec-202a75658889.png)